web/client/plugins/Map.jsx

```diff
@@ -117,17 +119,13 @@ const {handleCreationLayerError, handleCreationBackgroundError, resetMapOnInit}
  *
  * You can also specify a list if fonts that will be loaded before map is rendered (on willMount lyfe) and
  * if the shouldLoadFont is true
- * For each font you can specify an object describing the variation. The object can contain weight, style, and stretch properties.
- * If a property is not present it will default to normal.
+ * For each font you can specify the Fontname used in the `@font-face` declaration which must be included otherwise a notification error will appear:
```

Comment:
```suggestion
 * This font pre-load list is required if you're using canvas based mapping libraries (e.g. OpenLayers) and you need to show markers with symbols (e.g. Annotations). 
  * For each font you must specify the font name used in the `@font-face` inside the "fonts" array property. Note: the `@font-face` declaration must be present in css of the page, otherwise the font can not be loaded anyway. 
```